migrate.py

```diff
@@ -483,7 +483,11 @@ def rewrite_imports_in_fst(mod_fst, import_map, collection, spec, namespace, arg
             continue
         elif imp_src[1].value == 'module_utils':
             plugin_type = 'module_utils'
-            plugin_name = '/'.join(t.value for t in imp_src[token_length:])
+            try:
```

Comment:
First, I didn't get what it is that you're doing here.
May I suggest the following change to improve how it communicates the intent>
```suggestion
            plugin_name = '/'.join(t.value for t in imp_src[token_length:])
            with contextlib.suppress(AttributeError):
                plugin_name = f'{plugin_name}/{imp.targets[0].value}'
```
Plus let's avoid unnecessary duplication.